scipy/stats/stats.py

```diff
@@ -3748,52 +3754,56 @@ def alexandergovern(*args, nan_policy='propagate'):
 
     Examples
     --------
-        >>> from scipy.stats import alexandergovern
+    >>> from scipy.stats import alexandergovern
 
-        Here are some data on annual percentage rate of interest charged on
-        new car loans at nine of the largest banks in four American cities
-        taken from the National Institute of Standards and Technology's
-        ANOVA dataset.
+    Here are some data on annual percentage rate of interest charged on
+    new car loans at nine of the largest banks in four American cities
+    taken from the National Institute of Standards and Technology's
+    ANOVA dataset.
 
-        We use `alexandergovern` to test the null hypothesis that all cities
-        have the same mean APR against the alternative that the cities do not
-        all have the same mean APR. We decide that a confidence level of 95%
-        is required to reject the null hypothesis in favor of the alternative.
+    We use `alexandergovern` to test the null hypothesis that all cities
+    have the same mean APR against the alternative that the cities do not
+    all have the same mean APR. We decide that a sigificance level of 5%
+    is required to reject the null hypothesis in favor of the alternative.
 
-        >>> atlanta = [13.75, 13.75, 13.5, 13.5, 13.0, 13.0, 13.0, 12.75, 12.5]
-        >>> chicago = [14.25, 13.0, 12.75, 12.5, 12.5, 12.4, 12.3, 11.9, 11.9]
-        >>> houston = [14.0, 14.0, 13.51, 13.5, 13.5, 13.25, 13.0, 12.5, 12.5]
-        >>> memphis = [15.0, 14.0, 13.75, 13.59, 13.25, 12.97, 12.5, 12.25,
-        ...           11.89]
-        >>> alexandergovern(atlanta, chicago, houston, memphis)
-        AlexanderGovernResult(statistic=4.65087071883494,
-                              pvalue=0.19922132490385214)
+    >>> atlanta = [13.75, 13.75, 13.5, 13.5, 13.0, 13.0, 13.0, 12.75, 12.5]
+    >>> chicago = [14.25, 13.0, 12.75, 12.5, 12.5, 12.4, 12.3, 11.9, 11.9]
+    >>> houston = [14.0, 14.0, 13.51, 13.5, 13.5, 13.25, 13.0, 12.5, 12.5]
+    >>> memphis = [15.0, 14.0, 13.75, 13.59, 13.25, 12.97, 12.5, 12.25,
+    ...           11.89]
+    >>> alexandergovern(atlanta, chicago, houston, memphis)
+    AlexanderGovernResult(statistic=4.65087071883494,
+                          pvalue=0.19922132490385214)
 
-        The p-value is 0.1992, indicating a nearly 20% chance of observing
-        such an extreme value of the test statistic under the null hypothesis.
-        This exceeds 5%, so we do not reject the null hypothesis in favor of
-        the alternative.
+    The p-value is 0.1992, indicating a nearly 20% chance of observing
+    such an extreme value of the test statistic under the null hypothesis.
+    This exceeds 5%, so we do not reject the null hypothesis in favor of
+    the alternative.
     """
 
     args = _alexandergovern_input_validation(args, nan_policy)
 
+    if True in [(arg == arg[0]).all() for arg in args]:
```

Comment:
```suggestion
    np.any([(arg == arg[0]).all() for arg in args])
```
In the spirit of [Warren's other comment](https://github.com/scipy/scipy/pull/12873#discussion_r573512327).